testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/http/HttpInvocationInContainerTestCase.java

```diff
@@ -25,7 +28,10 @@
     @Deployment
     public static WebArchive deployment() {
         return ShrinkWrap.create(WebArchive.class, "http-test.war")
-                .addPackage(HttpInvocationInContainerTestCase.class.getPackage());
+                .addPackage(HttpInvocationInContainerTestCase.class.getPackage())
+                .addAsManifestResource(createPermissionsXmlAsset(
+                        new SocketPermission("localhost:8080", "connect,resolve")
```

Comment:
@hkalina The host and port can be overridden with some properties, so this is not guaranteed to be correct always. Do https://github.com/wildfly/wildfly/blob/master/testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/ejb/AuthenticationTestCase.java#L120 instead

Reply:
Thanks for tip! :)

Reply:
fixed
note: by keeping the constant inside of deployment() method there is no need for additional PropertyPermission (the deployment method run outside of AS only)